docs/releases/upcoming-features.md

```diff
@@ -12,41 +12,64 @@ Status                 | Details
 
 ### In Progress
 
-Details coming soon
+??? abstract "Node Telemetry"
 
-### Planning
+	Release  | V21 (Targeted)
+	---------|----------------------- 
+	Goal     | Provide methods for nodes to report block height, bandwidth caps, version numbers, node vendor versions and more.
```

Comment:
block ~~height~~ count? We usually associate height to chains. Unless we're including that in telemetry!